packages/network-controller/src/NetworkController.ts

```diff
@@ -78,6 +78,10 @@ export type NetworkDetails = {
   EIPS: {
     [eipNumber: number]: boolean;
   };
+  /**
+   * Indicates the availability of the network
+   */
+  status: NetworkStatus;
```

Comment:
With this new property we should remove this sentence from the JSDoc for this type above: "Currently only used to capture whether a network supports EIP-1559."